### ISSUE TITLE
icon in Unity app switcher not found

### ISSUE THREAD

**Issue by kw-at-oocl** [IsmaelMartinez/teams-for-linux#139]
In version 2.1, when switching apps in Unity (alt+tab), the icon shows as a question mark.  This was fixed by changing the below line.

File: /var/lib/snapd/desktop/applications/teams-for-linux_teams-for-linux.desktop
From:
`StartupWMClass=teams-for-linux`
To:
`StartupWMClass=teams for linux`

**Comment by IsmaelMartinez** [comment]
Thanks for reporting. Hopefully @julian-alarcon can have a look at this at some point. Feel free to see if you can fix it in the build in here and summit a PR. Cheers

**Comment by julian-alarcon** [comment]
Hi @kw-at-oocl . Which version are you using? The one from the Snap Store?
Try this command `sudo snap info teams-for-linux` and show us the output.

I'm using the current Snapstore version and have no issues, please also check of you have the latest version.

**Comment by kw-at-oocl** [comment]
Thanks for the reply.  Please find requested info below.  Also please note our Ubuntu is a bit dated:
NAME="Ubuntu"
VERSION="16.04.6 LTS (Xenial Xerus)"

$ snap info teams-for-linux
name:      teams-for-linux
summary:   Teams for Linux
publisher: –
license:   unset
description: |
  Unofficial Microsoft Teams client for Linux using Electron. It uses the Web App
   and wraps it as a standalone application using Electron.
commands:
  - teams-for-linux
refresh-date: 2 days ago, at 13:39 PDT
channels:
  stable:    –                           
  candidate: –                           
  beta:      0.2.0 2019-04-25  (4) 128MB -
  edge:      0.2.1 2019-05-14 (12) 128MB -
installed:   0.2.1            (x1) 128MB -
Thanks!

**Comment by julian-alarcon** [comment]
Hi @kw-at-oocl !

Can you update the Snap!? We released a new version recently. But before that, I think that it's better to remove the Snap, clean the conf of the snap located in ~/snap/teams-for-linux (just delete the folder after removing the snap)

**Comment by IsmaelMartinez** [comment]
@kw-at-oocl , any updates on this? Maybe a restart will do (Linux might be caching the ico or it might nit be finding it until a restart)

**Comment by kw-at-oocl** [comment]
Hi @julian-alarcon, unfortnately after updating to 0.3.0 the problem wasn't fixed.  In addition, the new icon isn't appearing in the Unity search now in addition to not appearing in alt-tab.  FWIW, the icon does show fine in Docky...

I rebooted as well as running `unity --replace &` thinking a reset of Unity might do it, but no luck. 
 
My fix described above still works to get the icon to show during alt-tab.  From what I gather, Unity uses the WM_CLASS setting to match the running app to an icon.  I used `xprop WM_CLASS` to see that the class name was reported without dashes as `WM_CLASS(STRING) = "teams for linux", "Teams for Linux"` which explains why my fix to remove the dashes from the desktop file worked.  I haven't poked around to see what can explain the new icon not showing in the Unity search...

On another sad note, we found a deal breaker for us.  The other office apps like Outlook and Onedrive can be accessed via the dots menu in the upper left corner.  This is a deal breaker for us to use the app, since the main reason we want to use the app is to support our pair programming practice and cutoff access to those other apps on our pairing workstations.

If this is something you feel can be addressed (preventing access to the other office apps) let me know and I'll open an issue for it.

Also feel free to close this issue as it may be related to our somewhat dated Ubuntu installation.

Thanks
